src/sentry/deletions/defaults/group.py

```diff
@@ -36,6 +89,15 @@ def get_child_relations(self, instance):
 
         relations.extend([ModelRelation(m, {"group_id": instance.id}) for m in model_list])
 
+        relations.extend(
```

Comment:
The event deletion would still run and thus try to delete the same nodes on nodestore. Would that make the nodestore deletion fail ?

Reply:
It is safe to delete the same node twice from nodestore. We are already performing 2 deletes from nodestore right now (which will soon both go away) - one triggered by event deletion https://github.com/getsentry/sentry/blob/master/src/sentry/deletions/defaults/event.py#L8-L15 and one by nodefield deletion https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/fields/node.py#L184).